src/main/scala/no/ndla/imageapi/controller/RawController.scala

```diff
@@ -25,7 +25,7 @@ trait RawController {
 
     val response404 = ResponseMessage(404, "Not found", Some("Error"))
     val response500 = ResponseMessage(500, "Unknown error", Some("Error"))
-    val cacheHeaders = Map("Cache-Control" -> "max-age=3600")
+    val staticAssetCacheHeaders = Map("Cache-Control" -> "public, max-age=31536000")
```

Comment:
Should be "public, max-age=31536000, immutable". See https://www.keycdn.com/blog/cache-control-immutable/ and https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control